src/main/js/controls/Graph/GraphConfig.js

```diff
@@ -164,7 +168,42 @@ export const validateContent = (content, input) => {
         throw new Error(errors.THROW_MSG_NON_UNIQUE_PROPERTY);
     }
 };
-
+/**
+ * Checks if panning is enabled or not
+ *
+ * @private
+ * @param {object} config - config object used by the graph.
+ * @returns {boolean} returns true of panning enabled else false.
+ */
+export const isPanningModeEnabled = (config) => {
+    if (config.pan !== undefined && config.pan.enabled) {
+        return true;
+    }
+    return false;
+};
+/**
+ * Used to set the clamp and transition when panning is enabled or not.
+ *
+ * @private
+ * @param {object} config - config object used by the graph.
+ * @returns {undefined} returns nothing
+ */
+export const settingsDictionary = (config) =>
```

Comment:
While we are considering moving `isPanningModeEnabled` function to `BaseConfig` we **cant** move this function. 
If you are considering including the dateline property here. (Timeline doesnt have a dateline).